cypress/e2e/tests/pages/manager/machine-sets.spec.ts

```diff
@@ -102,7 +102,9 @@ describe('MachineSets', { testIsolation: 'off', tags: ['@manager', '@adminUser']
 
     // check details page
     machineSetsPage.list().details(this.machineSetName, 2).find('a').click();
-    cy.contains('secretName2').should('be.visible');
+    cy.contains('secretName2')
+      .scrollIntoView()
```

Comment:
The new masthead takes up more vertical space. I needed to scroll the element into view so the visibility check would work.